include/Map.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "Tile.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using tileStdMap = std::map<std::pair,tile::Tile>;
+using entityStdMap = std::map<uint32_t,entity::Entity>;
+
+class Map {
+    private:
+        uint16_t id_;
+        uint8_t currentType_;
+        std::string mapName_;
+        std::string bitmapName_;
+        tileStdMap tiles_; //<Coordinates,tile>
```

Comment:
Why have you done this: `using tileStdMap = std::map<std::pair,tile::Tile>;` if you end up adding the type as a comment anyways :)

```suggestion
        tileStdMap tiles_;
```